### DISCUSSION TITLE
AAX Export in Ordner

### DISCUSSION THREAD

**Discussion by enno22** [audiamus/BookLibConnect#35] [Ideas]
Hallo, habe einen kleinen Verbesserungsvorschlag: Wäre es möglich beim AAX Export die AAX Datei und die beiden Zusatzdateien mit Kapitel Informationen usw. in einen jeweiligen eigenen Ordner zu exportieren ?
VG
Enno

**Comment by audiamus** [comment]
Das hätte aber den Nachteil, dass man bei der Weiterverarbeitung mit AAX Audio Converter - und dafür ist der Export ja gedacht - keine Mehrfachselektion beim Einlesen mehr nutzen kann, weder per Datei-Dialog noch per Drag & Drop. 

Mir ist schon klar, dass die Zuordnung zwischen Audio-Datei und den zugehörigen Meta-Dateien für den menschlichen Leser mühsam ist, aber AAX Audio Converter hat die Probleme natürlich nicht.
